### ISSUE TITLE
GeoJSON LineStrings supported?

### ISSUE THREAD

**Issue by TheMapSmith** [uber/kepler.gl#97]
Hello,

I'm trying to visualize GeoJSON LineStrings of aircraft flight paths and nothing happens. I'm prompted to pick Lat0/Lng0 and Lat1/Lng0, but that makes it seem like it only supports "lines" with a start/finish location rather than multi-vertex linestrings. Is that correct? Are traditional line strings not supported at this time?

**Comment by heshan0131** [comment]
Hi @TheMapSmith , `LineStrings` are supported. You should include it in a `FeatureCollection` as a `Feature`. You can take a look at [Geojson Files](https://github.com/uber/kepler.gl/blob/master/docs/b-kepler-gl-workflow/a-add-data-to-the-map.md#for-geojson-files) part of the user guide

**Comment by TheMapSmith** [comment]
Hi @heshan0131, I realized that it wasn't working because my GeoJSON was invalid. However, once my file was valid the rendering was very strange. 

<img width="532" alt="screen shot 2018-06-05 at 10 36 30 am" src="https://user-images.githubusercontent.com/8357118/40982945-825e31c6-68ac-11e8-9541-dec6c7e7aa7c.png">

It automatically selected a "Polygon" type. Is that the intended behavior?

**Comment by TheMapSmith** [comment]
Here is an ephemeral link to the file if you want to test it out 

300MB GeoJSON: 

https://wetransfer.com/downloads/8213d2abbb07200feb5a4b175338d87920180605144346/4b6ff3e7f5a21f61290d928c130cedc420180605144346/fe00c9 

The link will expire in 7 days

**Comment by yiyaya66** [comment]
@TheMapSmith we are currently in the progress of fixing this.

**Comment by patqzzzzzz** [comment]
Hi,

I have faced a similar issue when I was attempting to visualise the bus routes in Singapore. 

![singapore_bus_routes_100perc](https://user-images.githubusercontent.com/39853234/42016028-708eaa24-7adc-11e8-95ab-ab8f3f00e791.png)

There were random lines poking out of and towards nowhere.
I tried to look through the geojson file and there doesn't seem to be any issues with the coordinates or the formatting of the features.

In an attempt of zooming into the map, the lines became shortened and eventually disappeared/became negligible. 

![singapore_bus_routes_zoomed](https://user-images.githubusercontent.com/39853234/42016113-cc311e02-7adc-11e8-8b11-0d88ddc5b0bc.png)

Following that, we attempted to zoom out of the browser window and the map appeared to be fine.

![singapore_bus_routes_25perc](https://user-images.githubusercontent.com/39853234/42016128-e6fd5f20-7adc-11e8-9e5d-77c4f983ffaa.png)

Hope this is helpful! Thanks!
